cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTask.java

```diff
@@ -55,10 +56,11 @@
     VALID_TRANSFER.put(ABORTED, Collections.emptySet());
   }
 
-  public ExecutionTask(long executionId, ExecutionProposal proposal) {
-    this.executionId = executionId;
-    this.proposal = proposal;
-    this._state = State.PENDING;
+  public ExecutionTask(long executionId, ExecutionProposal proposal, TaskType type) {
```

Comment:
Should we add a JavaDoc? Maybe we can move single line comments of private variable names to params in here.